migration/migrator/data/course_tables.sql

```diff
@@ -417,6 +417,21 @@ CREATE TABLE peer_assign (
     user_id character varying NOT NULL
 );
 
+--
+-- Name: peer_feedback; Type: TABLE; Schema: public; Owner: -
+--
+
+CREATE TABLE peer_feedback (
+    pf_id integer NOT NULL,
+    grader_id character varying(255) NOT NULL,
+    user_id character varying(255),
+    team_id character varying(255),
+    g_id character varying(255) NOT NULL,
+    feedback_full character varying(255),
+    feedback_id character varying(255),
```

Comment:
It looks like you dropped one of these in the migration but forgot to do it here.

Reply:
Dropped a column to match the migration file